dockerfiles/entrypoint.sh

```diff
@@ -1,20 +1,16 @@
 #!/bin/bash
 
 set -e
-/usr/sbin/sshd
 
 if [ "$#" = 0 ]; then
 	tail -F -n0 /etc/hosts && wait
 elif [ "$1" = "controller" ]; then
 	redis-server &
-	splunk_eventgen service --role controller &
-	tail -F -n0 /etc/hosts && wait
+	splunk_eventgen service --role controller
 elif [ "$1" = "server" ]; then
-	splunk_eventgen service --role server &
-	tail -F -n0 /etc/hosts && wait
```

Comment:
Because of the use cases internally, we would rather prefer to keep the container running despite the Eventgen process dies.